types/ClassFile/constants.ts

```diff
@@ -0,0 +1,106 @@
+import {CONSTANT_TAG} from "../../constants/ClassFile/constants"
+
+export interface CONSTANT_Class_info {
```

Comment:
Shall we standardize the names of interfaces in this file with other files by using Pascal case? For example, use `ConstantClassInfo` and `ConstantFieldrefInfo` instead.

Reply:
The constant names are directly taken from oracle docs (e.g. `Class_info`), and I prepended `CONSTANT_` to all of them. I am not sure if there is an easy way to rename all of them, would you know anything about refactoring names?

Reply:
Ya, I can help to change all of them if you also agree on this ([reference](https://basarat.gitbook.io/typescript/styleguide)).

Reply:
Sure, go ahead, thanks!